earth_enterprise/src/fusion/tools/gee_test/status.sh

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/python
+#!/bin/bash -eu
 #
-# Copyright 2017 Google Inc.
+# Copyright 2017 Open GEE Authors
```

Comment:
I couldn't remember what we had decided on before. I looked at other Google-sponsored projects and they were using Authors. But, we can use Contributors. I don't have a strong preference.